### DISCUSSION TITLE
Google Oauth redirects to localhost

### DISCUSSION THREAD

**Discussion by Alex-Giaquinto** [outline/outline#6430] [Self Hosting]
I set up a self hosted instance of outline. Install went smoothly and when I traveled to the webpage I was brought to the home screen with the continue with google option. When I click google it redirects to localhost:3000/auth/google and I receive an unable to connect error. I followed the docs accordingly. Any ideas?

**Comment by tommoor** [comment]
It sounds like your environment contains the wrong value for `URL`

**Comment by Alex-Giaquinto** [comment]
Ah you are right that definetly makes sense! Thank you @tommoor !! Would I also have to change the port config from 3000 to 443 if I am using a reverse proxy (Nginx)?

**Reply by tommoor** [reply]
The `PORT` config would reflect the port that outline communicates to nginx on – likely it stays at `3000`

**Reply by Alex-Giaquinto** [reply]
@tommoor I was able to get everything to work. My only question is now how do I make specific users admins?
